commands/drive.js

```diff
@@ -0,0 +1,39 @@
+module.exports = function (bb8) {
+
+  //variation of the roll command to enable input from keyboard
+  //useful when it you get commands from the Internet or just to play with
+  //BB-8 using nodejs
+  console.log('Drive mode enabled');
+  console.log('Press ctrl+c to exit');
+
+  var stdin = process.stdin;
+  stdin.setRawMode(true);
+  stdin.resume();
+  stdin.setEncoding('utf8');
+
+  stdin.on('data', function(key){
+    bb8.color('#000000');
+    if (key == '\u001B\u005B\u0041') {
```

Comment:
These key references are pretty ambiguous and its not clear what they are. Maybe we could use the [keypress](https://github.com/TooTallNate/keypress) library instead to make it clearer which key is which.